radio/src/lua/api_lcd.cpp

```diff
@@ -123,21 +123,57 @@ static int luaLcdDrawLine(lua_State *L)
 
 #if !defined(COLORLCD)
 /*luadoc
-@function lcd.getLastPos()
+@function lcd.getLastPos() - DEPRECATED
 
 Returns the last x position from previous output
 
 @retval number (integer) x position
 
-@notice Only available on Taranis
+@notice Only available on Taranis, please use lcd.getRightPos() instead
 
-@status current Introduced in 2.0.0
+@status current Introduced in 2.0.0, deprecated in 2.2
 */
 static int luaLcdGetLastPos(lua_State *L)
 {
-  lua_pushinteger(L, lcdLastPos);
+  lua_pushinteger(L, lcdRightPos);
+  TRACE("WARNING : lcd.getLastPos() is deprecated, please move to lcd.getLeftPos() or lcd.getRightPos()");
```

Comment:
As previously discussed, this completely spams the console, very nasty.  Warn once if you need to, but don't make the console otherwise useless (also potentially crash on smaller/slower computers).  It is not going to be a "simple replace" of getLastPos() for all users due to numerous backward compatibility issues.